doc/source/ray-debugging.rst

```diff
@@ -0,0 +1,174 @@
+Ray Debugging
+=============
+
+Ray has a built in debugger that allows you to debug your distributed applications.
+
+.. note::
+
+    It is currently an experimental feature and under active development. Interfaces are subject to change.
+
+Getting Started
+---------------
+
+Take the following example that is modified from the Ray README:
+
+.. code-block:: python
+
+    import ray
+    ray.init()
+
+    @ray.remote
+    def f(x):
+        ray.util.pdb.set_trace()
```

Comment:
Is there a docstring for this?

Reply:
If so, can you please provide it? 

This will be much more useful for users with an IDE.

Reply:
Thanks, I added a docstring!